layouts/partials/navigators/theme-selector.html

```diff
@@ -0,0 +1,12 @@
+<li class="nav-item dropdown">
+<!-- This is a 'hack' for initializing the color scheme selection. See /js/darkmode.js -->
+<div id="themeInitialization" style="display: none;"
+    default-theme=" {{ site.Params.darkTheme.default }} "></div>
+<a class="nav-link dropdown-toggle"  href="#" id="themeSelector" role="button"
```

Comment:
Can we use icon instead of text? It will differentiate it from rest of the menus. Probably, something like this: https://tailwindcss.com/